spec/unit/type/user_spec.rb

```diff
@@ -351,13 +351,20 @@ def self.instances; []; end
       @user = described_class.new(:name => 'foo', :comment => @value)
     end
 
-    it "should be converted to ASCII_8BIT for ruby 1.9 / 2.0", :if => RUBY_VERSION < "2.1.0" && String.method_defined?(:encode) do
-      expect(@user[:comment].encoding).to eq(Encoding::ASCII_8BIT)
-      expect(@user[:comment]).to eq(@value.force_encoding(Encoding::ASCII_8BIT))
-    end
-
-    it "must not be converted for ruby >= 2.1", :if => RUBY_VERSION >= "2.1.0" do
-      expect(@user[:comment].encoding).to eq(Encoding::UTF_8)
+    it "should define the #insync?(is,should) method" do
+      comment_property = @user.properties.find {|p| p.name == :comment}
+      # class#instance_methods(false) returns methods only defined on the instance and no ancestors
+      expect(comment_property.class.instance_methods(false).include?(:insync?)).to be_truthy
+    end
+
+    describe "#insync" do
+      it "should delegate to the provider's #comments_insync? method if defined" do
```

Comment:
I would add this case in, I don't think there is anything wrong with checking for super based on specific parameters